pkg/artifactory/resource_artifactory_local_repository_test.go

```diff
@@ -90,7 +89,6 @@ func TestAccLocalRepository_full(t *testing.T) {
 					resource.TestCheckResourceAttr("artifactory_local_repository.terraform-local-test-repo-full", "calculate_yum_metadata", "false"),
 					resource.TestCheckResourceAttr("artifactory_local_repository.terraform-local-test-repo-full", "yum_root_depth", "0"),
 					resource.TestCheckResourceAttr("artifactory_local_repository.terraform-local-test-repo-full", "docker_api_version", "V2"),
-					resource.TestCheckResourceAttr("artifactory_local_repository.terraform-local-test-repo-full", "force_nuget_authentication", "true"),
```

Comment:
You can't enable `forceNugetAuthentication` on a npm repository. I'm not sure how this ever worked or why this was added.

Reply:
This is what I meant by "repository +"